lib/structures/Message.js

```diff
@@ -291,8 +292,15 @@ class Message extends Base {
                 };
             });
         }
-        if(data.stickers !== undefined) {
-            this.stickers = data.stickers;
+        if(data.sticker_items !== undefined) {
+            this.sticker_items = data.sticker_items.map((sticker) => {
```

Comment:
camelCase (also update JSDoc ordering when you change this)

Reply:
Well I preferred camelCase too, but you said `sticker_items` so I didn't bother questioning it haha, thanks & will do 👍🏻